src/pages/instances/InstanceDetailActions.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC, cloneElement } from "react";
+import DeleteInstanceBtn from "./actions/DeleteInstanceBtn";
+import { LxdInstance } from "types/instance";
+import MigrateInstanceBtn from "./actions/MigrateInstanceBtn";
+import CreateImageFromInstanceBtn from "./actions/CreateImageFromInstanceBtn";
+import DuplicateInstanceBtn from "./actions/DuplicateInstanceBtn";
+import { ContextualMenu } from "@canonical/react-components";
+
+interface Props {
+  instance: LxdInstance;
+  project: string;
+  isLoading: boolean;
+  isLargeScreen: boolean;
+}
+
+const InstanceDetailActions: FC<Props> = ({
+  instance,
+  project,
+  isLoading,
+  isLargeScreen,
+}) => {
+  const hasMediumScreenClass = isLargeScreen ? "p-contextual-menu__link" : "";
```

Comment:
```suggestion
  const classname = isLargeScreen ? "p-contextual-menu__link" : "";
```
I think this better describes the variable contents.